pkg/assembler/clients/helpers/assembler.go

```diff
@@ -177,6 +182,75 @@ func hasSourceAt(ctx context.Context, client graphql.Client, hsa assembler.HasSo
 	return err
 }
 
+func ingestVex(ctx context.Context, client graphql.Client, vis []assembler.VexIngest) error {
+	for _, vi := range vis {
+		if vi.CVE != nil && vi.GHSA != nil && vi.OSV != nil {
```

Comment:
we should add a check that only one is specified for the vulnerabilities. A helper function [here](https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/helpers/assembler.go#L248) can be used.